tests/transport/compliance.py

```diff
@@ -129,3 +129,15 @@ def test_cached_request(self, server):
         new_time = new_response.headers.get("Time")
         
         assert request_time == new_time, "{} and {} are not equal".format(request_time, new_time)
+    
+    def test_expired_cached_request(self, server):
+        request = self.make_cached_request()
+        response = request(url=server.url + "/cache", method="GET")
+        request_time = response.headers.get("Time")
+        
+        time.sleep(_MAX_AGE)
```

Comment:
Is there a way to mock the time instead of explicitly having the program sleep?

Reply:
found a nice way to do this with the freezegun library. essentially you freeze the time at the start of the test and increment the time as needed throughout the test to mimic the .sleep() call. Used a mock object to replace time.sleep() with this behavior in the tests